Adafruit_INA219.cpp

```diff
@@ -102,8 +81,8 @@ void Adafruit_INA219::setCalibration_32V_2A(void)
   // setting the VBUS_MAX to 16V instead of 32V, etc.
 
   // VBUS_MAX = 32V             (Assumes 32V, can also be set to 16V)
-  // VSHUNT_MAX = 0.32          (Assumes Gain 8, 320mV, can also be 0.16, 0.08, 0.04)
-  // RSHUNT = 0.1               (Resistor value in ohms)
+  // VSHUNT_MAX = 0.32          (Assumes Gain 8, 320mV, can also be 0.16, 0.08,
+  // 0.04) RSHUNT = 0.1               (Resistor value in ohms)
```

Comment:
This is formatted a little confusingly; can you have the gain values continue on a line above?

Reply:
you are right, I'm using `clang-format` and It sometime do things like this :)